### ISSUE TITLE
[V3] Implement new Concept Exercise: Comparison and Conditionals

### ISSUE THREAD

**Issue by junedev** [exercism/javascript#989]
## Getting started
**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md)
- [What are concepts and how they are structured?](https://github.com/exercism/docs/blob/main/anatomy/tracks/concepts.md)
- [What are concept exercises and how they are structured?](https://github.com/exercism/docs/blob/main/anatomy/tracks/concept-exercises.md)

Please also watch the following video:
- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

See the documentation _above_ (general documentation), as well as [How to implement a Concept Exercise in JavaScript](https://github.com/exercism/javascript/blob/main/reference/implementing-a-concept-exercise.md).

## Goal
The goal of this exercise is to teach the student how to compare values in Javascript and then use that in conditional statements.

## Concepts
The following concept pages need to be created. You can combine the two introduction.md files from the concepts and use that as introduction.md for the concept exercise. No need to create different content at this point.
* `comparison`
* `conditionals`

## Learning Objectives
In the concepts the student should learn about the following operators and statements and then practice them in the concept exercise.

### Comparison
* `<`, `>`, `>=`, `<=` for numbers and strings
* strict equality (`===`)
* not equal (`!==`)

### Conditionals
* `if(){}`
* `if(){} else{}`
* `if(){} else if(){} else{}`

## Out of Scope
* Loose equality `==` and truthy/falsy (will be taught once the student learned about type coercion)
* Comparing values of different types
* Shallow/deep comparison of objects
* ternary operator `x ? y : z`
* `switch`, `for`

## About.md of the Concept
If there are no pitfalls, advanced usages etc. that you would like to point out, the about.md file can be a copy of the introduction.md of the concept.

For the `conditionals` concept here some ideas what to mention in the about.md file.
* multiple `else if`
* if statements without a block to execute just one thing (e.g., `if(...) return ...`)

## Prerequisites
* `basics`
* `booleans`
* `numbers`
* `strings`

When the students start this exercise they only know some basics about the primitive types. Keep that in mind when designing the exercise. See [this list](https://github.com/exercism/javascript/issues/951#issuecomment-778834953) for details on the learning curve we are aiming for.

## Exercise Idea
[Julia Conditionals Exercise](https://github.com/exercism/julia/blob/main/exercises/concept/vehicle-purchase/.docs/instructions.md) could be used as template

## Help
You can choose to do this solo-style, or collaborate with multiple people on this. The suggested approach is to 
1. **First accept this issue by saying "I'd like to work on this"** (you will not get a response, just go with it) and optionally request that someone works with you (and wait for a second person to accept your request).
2. Use this issue to write down what you need (collect reference material) and discuss, if necessary, what to write and what not to write.
3. Draft a PR and request the feedback from at least one other JavaScript member (you can ping the #maintaining-javascript channel in Slack).

**Comment by junedev** [comment]
I will work on this.

**Comment by SleeplessByte** [comment]
FYI, I am expecting `<repo-root>/concepts/comparison` to exist after this PR as I am linking to it :)

**Comment by junedev** [comment]
FYI a quick status update: I am done, I posted the PR as draft, I still need to resolve some conflicts and give it a self-review

**Comment by SleeplessByte** [comment]
I'll review it once you undraft it :)
